gcn/notices/fermi/gbm/alert.example.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/fermi/gbm/alert.schema.json",
+  "additional_info": "Lightcurve and other quicklook files will not be created until about 15 minutes after the trigger.",
+  "alert_datetime": "2024-08-24T02:14:54",
+  "alert_tense": "current",
+  "alert_type": "initial",
+  "data_archive_page": "http://heasarc.gsfc.nasa.gov/FTP/fermi/data/gbm/triggers/2024/bn240812094",
+  "detectors_status": [
+    "triggered",
+    "on",
+    "triggered",
+    "triggered",
+    "on",
+    "on",
+    "triggered",
+    "on",
+    "on",
+    "on",
+    "on",
+    "on",
+    "on",
+    "on"
+  ],
+  "event_name": ["GRB240824A"],
+  "human_in_the_loop": false,
+  "id": ["745121685", "bn240812094"],
+  "instrument": "GBM",
+  "latitude": 23.83,
+  "light_curve_url": "http://heasarc.gsfc.nasa.gov/FTP/fermi/data/gbm/triggers/2024/bn240812094/quicklook/glg_lc_medres34_bn240812094.gif",
```

Comment:
These don't need to be alphabetical, and it seems strange to separate latitude and longitude.  In general, these examples should be in some logical order.

Reply:
Alphabetical order helps bookkeeping. I'd like to keep it this way.